src/containers/AccountModal/AccountModal.tsx

```diff
@@ -60,6 +61,22 @@ const AccountModal = () => {
     }
   }, [viewParam, loading, isPublicView, user, toLogin]);
 
+  // handle websocket notifications
+  useEffect(() => {
```

Comment:
I understand why this is in the Account Modal, but it might be nicer to put closer to / in the Root.tsx since it is such a 'global' thing.

Reply:
The dilemma was between AppRoutes and User, but I guess in the user feels more isolated and all the needed stuff was already there.. It can go in the Root as well, but we need to add user store checks, navigation imports,.. etc.

Reply:
If its a hook then all of those dependencies can go into the hook

Reply:
updated with hook